ROMFS/px4fmu_common/init.d/airframes/2106_albatross

```diff
@@ -35,11 +31,60 @@ param set-default FW_WR_IMAX 0.8
 param set-default FW_WR_P 1
 param set-default FW_W_RMAX 0
 
-# set disarmed value for the ESC
-param set-default PWM_MAIN_DISARM 1000
+param set-default CA_AIRFRAME 1
+param set-default CA_ROTOR_COUNT 1
+param set-default CA_ROTOR0_PX 0.3
+param set-default CA_SV_CS_COUNT 4
+param set-default CA_SV_CS0_TYPE 1
+param set-default CA_SV_CS0_TRQ_R -0.5
+param set-default CA_SV_CS1_TRQ_R 0.5
+param set-default CA_SV_CS1_TYPE 2
+param set-default CA_SV_CS2_TRQ_P 1.0
+param set-default CA_SV_CS2_TYPE 3
+param set-default CA_SV_CS3_TRQ_Y 1.0
+param set-default CA_SV_CS3_TYPE 4
+
+
+param set-default CA_AIRFRAME 1
+param set-default CA_ROTOR_COUNT 1
+param set-default CA_SV_CS_COUNT 7
+param set-default CA_SV_CS0_TRQ_R -0.5000
+param set-default CA_SV_CS0_TYPE 1
+param set-default CA_SV_CS1_TRQ_R 0.5000
+param set-default CA_SV_CS1_TYPE 2
+param set-default CA_SV_CS2_TRQ_P 0.5000
+param set-default CA_SV_CS2_TRQ_Y 0.5000
+param set-default CA_SV_CS2_TYPE 7
+param set-default CA_SV_CS3_TRQ_P 0.5000
+param set-default CA_SV_CS3_TRQ_Y -0.5000
+param set-default CA_SV_CS3_TYPE 8
+param set-default CA_SV_CS4_TRQ_Y 1.0000
+param set-default CA_SV_CS4_TYPE 12
```

Comment:
The wheel would need to be separated from the torque channel as otherwise the allocation of yaw torque is no longer correct, no?

Reply:
It should be a matter of scale, but certainly something to check.
I agree it should be separate, it could eventually be a separate yaw wheel controller, and for now we could pass through the yaw control signal as output function?

Reply:
>and for now we could pass through the yaw control signal as output function?

Yep that's prob the best we can do right now. It falls into the same category as flaps, spoilers, airbrakes, landing gear, but for now it's fine to pass it in with the yaw control signal as long as it's handles separately from the rudder.

Reply:
Added the steering wheel as a separate type in be9fd21d1e60e00a2cae339293d6beb9de16e69f. It's in the category "control surfaces" for now, we may want to add a separate category for it. And also think about how the category would be names for rovers, and if there we also want to move away from feeding the "yaw" command through a matrix.
Also added a "Single Channel Aileron" type, as it is quite common to have the same control signal for both ailerons.